scrapy_jsonschema/item.py

```diff
@@ -1,27 +1,25 @@
 import re
-import six
+
 from jsonschema import (
     Draft3Validator,
     Draft4Validator,
     Draft6Validator,
     Draft7Validator,
     FormatChecker,
+    draft3_format_checker,
+    draft4_format_checker,
+    draft6_format_checker,
+    draft7_format_checker,
```

Comment:
:lipstick:

```suggestion
    draft3_format_checker,
    Draft3Validator,
    draft4_format_checker,
    Draft4Validator,
    draft6_format_checker,
    Draft6Validator,
    draft7_format_checker,
    Draft7Validator,
    FormatChecker,
```

I think this is the most common approach, but I have no source to support this believe.

Reply:
This was the result of a line sorting on one of my machines, I just didn't pay much attention to the outcome. Now I'm doing it on my other computer, and I'm getting what you suggest. I suppose I have the sorting configured differently when it comes to uppercase and lowercase chars :shrug: 
Thanks!